diff --git a/src/rp2_common/pico_btstack/CMakeLists.txt b/src/rp2_common/pico_btstack/CMakeLists.txt
index c622cac..2078825 100644
--- a/src/rp2_common/pico_btstack/CMakeLists.txt
+++ b/src/rp2_common/pico_btstack/CMakeLists.txt
@@ -53,6 +53,7 @@ if (EXISTS ${PICO_BTSTACK_PATH}/${BTSTACK_TEST_PATH})
         ${PICO_BTSTACK_PATH}/src/hci_cmd.c
         ${PICO_BTSTACK_PATH}/src/hci_dump.c
         ${PICO_BTSTACK_PATH}/src/hci_event.c
+        ${PICO_BTSTACK_PATH}/src/hci_event_builder.c
         ${PICO_BTSTACK_PATH}/src/l2cap.c
         ${PICO_BTSTACK_PATH}/src/l2cap_signaling.c
         ${PICO_BTSTACK_PATH}/3rd-party/md5/md5.c
@@ -90,7 +91,7 @@ if (EXISTS ${PICO_BTSTACK_PATH}/${BTSTACK_TEST_PATH})
         ${PICO_BTSTACK_PATH}/src/ble/gatt-service/device_information_service_server.c
         ${PICO_BTSTACK_PATH}/src/ble/gatt-service/device_information_service_client.c
         ${PICO_BTSTACK_PATH}/src/ble/gatt-service/heart_rate_service_server.c
-        ${PICO_BTSTACK_PATH}/src/ble/gatt-service/hids_client.c
+        ${PICO_BTSTACK_PATH}/src/ble/gatt-service/hids_host.c
         ${PICO_BTSTACK_PATH}/src/ble/gatt-service/hids_device.c
         ${PICO_BTSTACK_PATH}/src/ble/gatt-service/nordic_spp_service_server.c
         ${PICO_BTSTACK_PATH}/src/ble/gatt-service/ublox_spp_service_server.c
@@ -318,7 +319,7 @@ if (EXISTS ${PICO_BTSTACK_PATH}/${BTSTACK_TEST_PATH})
         set_source_files_properties(
             ${PICO_BTSTACK_PATH}/src/ble/att_server.c
             ${PICO_BTSTACK_PATH}/src/ble/gatt-service/device_information_service_server.c
-            ${PICO_BTSTACK_PATH}/src/ble/gatt-service/hids_client.c
+            ${PICO_BTSTACK_PATH}/src/ble/gatt-service/hids_host.c
             ${PICO_BTSTACK_PATH}/src/btstack_util.c
             ${PICO_BTSTACK_PATH}/src/btstack_crypto.c
             ${PICO_BTSTACK_PATH}/src/classic/a2dp.c
@@ -381,7 +382,7 @@ if (EXISTS ${PICO_BTSTACK_PATH}/${BTSTACK_TEST_PATH})
             COMPILE_OPTIONS "-Wno-unused-parameter;-Wno-sign-compare"
             )
         set_source_files_properties(
-            ${PICO_BTSTACK_PATH}/src/ble/gatt-service/hids_client.c
+            ${PICO_BTSTACK_PATH}/src/ble/gatt-service/hids_host.c
             PROPERTIES
             COMPILE_OPTIONS "-Wno-cast-qual;-Wno-null-dereference"
             )
