diff --git a/src/class/hid/hid_host.h b/src/class/hid/hid_host.h
index 9681c704b..ad055105e 100644
--- a/src/class/hid/hid_host.h
+++ b/src/class/hid/hid_host.h
@@ -28,6 +28,7 @@
 #define _TUSB_HID_HOST_H_
 
 #include "hid.h"
+#include "host/usbh.h"
 
 #ifdef __cplusplus
 extern "C" {
diff --git a/src/host/usbh.c b/src/host/usbh.c
index a2994cde7..052f17d1c 100644
--- a/src/host/usbh.c
+++ b/src/host/usbh.c
@@ -176,7 +176,7 @@ static usbh_class_driver_t const usbh_class_drivers[] = {
   },
   #endif
 
-  #if CFG_TUH_HID
+  #if CFG_TUH_HID_NO
   {
       .name       = DRIVER_NAME("HID"),
       .init       = hidh_init,
