diff --git a/src/l2cap.c b/src/l2cap.c
index 9ffa9c102..d78b59a39 100644
--- a/src/l2cap.c
+++ b/src/l2cap.c
@@ -2455,7 +2455,11 @@ static void l2cap_handle_remote_supported_features_received(l2cap_channel_t * ch
 
         // incoming: assert security requirements
         channel->state = L2CAP_STATE_WAIT_INCOMING_SECURITY_LEVEL_UPDATE;
+#if 0
         if (channel->required_security_level <= gap_security_level(channel->con_handle)){
+#else
+        if (1) {
+#endif
             l2cap_handle_security_level_incoming_sufficient(channel);
         } else {
             // send connection pending if not already done
